lib/java/com/google/android/material/lists/viewholders/SingleLineItemViewHolder.java

```diff
@@ -14,33 +14,37 @@
  * limitations under the License.
  */
 
-package com.google.android.material.lists;
+package com.google.android.material.lists.viewholders;
 
 import com.google.android.material.R;
+import com.google.android.material.lists.MaterialViewHolder;
 
 import androidx.annotation.NonNull;
-import androidx.recyclerview.widget.RecyclerView.ViewHolder;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Context;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 /** A simple single line list item. */
-public class SingleLineItemViewHolder extends ViewHolder {
+public class SingleLineItemViewHolder extends MaterialViewHolder {
+
+  public ImageView icon;
+  public TextView text;
 
-  public final ImageView icon;
-  public final TextView text;
+  public SingleLineItemViewHolder(@NonNull View itemView) {
+    super(itemView);
 
-  public SingleLineItemViewHolder(@NonNull View view) {
-    super(view);
     this.icon = itemView.findViewById(R.id.mtrl_list_item_icon);
     this.text = itemView.findViewById(R.id.mtrl_list_item_text);
+
   }
 
-  @NonNull
-  public static SingleLineItemViewHolder create(@NonNull ViewGroup parent) {
-    return new SingleLineItemViewHolder(LayoutInflater.from(parent.getContext())
-        .inflate(R.layout.material_list_item_single_line, parent, false));
+  public SingleLineItemViewHolder(@NonNull ViewGroup parent){
```

Comment:
Space missing between the parameter declaration and the constructor body:

```suggestion
  public SingleLineItemViewHolder(@NonNull ViewGroup parent) {
```